### PR TITLE
can make project rename part of clone

### PR DESCRIPTION
**Pull request by jnothman**

### CODE REVIEW

**Comment by vighneshbirodkar** [comment]
@jnothman
Thanks once again. With your <PROJECT-NAME> addition I have a few more suggestions. Can I suggest them here or would you prefer it if I merged this and did it myself?

**Comment by jnothman** [comment]
I don't have a problem with you merging and doing yourself, but if you want to discuss, here is perhaps the best venue.

File: README.md [hunk 1]
```diff
@@ -43,13 +43,13 @@ tests under `sklstub/tests` which can be run using `nosetests`.
...
-$ git clone https://github.com/vighneshbirodkar/sklearn-stub.git
+$ git clone https://github.com/vighneshbirodkar/sklearn-stub.git <PROJECT-NAME>
 ```
-You should rename the `sklearn-stub` folder to the name name of your project.
+replacing `<PROJECT-NAME>`.
 To host the project on Github, visit https://github.com/new and create a new
 repository. To upload your project on Github execute
 ```shell
-$ git remote add origin https://github.com/username/project-name.git
+$ git remote add origin https://github.com/username/<PROJECT-NAME>.git
```
Comment (jnothman) [inline README.md:52]: should probably be `<USERNAME>` for consistency...

File: README.md [hunk 2]
```diff
@@ -43,13 +43,13 @@ tests under `sklstub/tests` which can be run using `nosetests`.
 ### 1. Cloning
 Clone the project into your computer by executing
 ```shell
-$ git clone https://github.com/vighneshbirodkar/sklearn-stub.git
+$ git clone https://github.com/vighneshbirodkar/sklearn-stub.git <PROJECT-NAME>
 ```
-You should rename the `sklearn-stub` folder to the name name of your project.
+replacing `<PROJECT-NAME>`.
 To host the project on Github, visit https://github.com/new and create a new
```
Comment (vighneshbirodkar) [inline README.md:49]: We should also emphasize that the user needs to create a new github repo called `<PROJECT-NAME>`

**Pull request closed without merging**